nautobot/extras/jobs.py

```diff
@@ -381,6 +382,21 @@ def deserialize_data(cls, data):
 
         return return_data
 
+    def validate_data(self, data):
+        vars = self._get_vars()
+
+        if type(data) is not dict:
```

Comment:
Isn’t an `isinstance` check more the more idiomatic way?

Reply:
Yes, good call. We do want to converge on `isinstance` throughout the code (#591).

Reply:
Thank you, I moved to `isinstance`!